uniffi_bindgen/src/bindings/kotlin/templates/MapTemplate.kt

```diff
@@ -2,15 +2,14 @@
 {%- let value_type_name = value_type|type_name %}
 public object {{ ffi_converter_name }}: FfiConverterRustBuffer<Map<{{ key_type_name }}, {{ value_type_name }}>> {
     override fun read(buf: ByteBuffer): Map<{{ key_type_name }}, {{ value_type_name }}> {
-        // TODO: Once Kotlin's `buildMap` API is stabilized we should use it here.
-        val items : MutableMap<{{ key_type_name }}, {{ value_type_name }}> = mutableMapOf()
-        val len = buf.getInt()
-        repeat(len) {
-            val k = {{ key_type|read_fn }}(buf)
-            val v = {{ value_type|read_fn }}(buf)
-            items[k] = v
+        return buildMap<{{ key_type_name }}, {{ value_type_name }}> {
+            val len = buf.getInt()
```

Comment:
There is a `buildMap` variant that takes a `capacity` parameter and should be more performant, maybe the `len` value could be used for that?

```kotlin
val len = buf.getInt()
return buildMap<{{ key_type_name }}, {{ value_type_name }}>(len) {
```

Reply:
Thank you, didn't notice that. Fixed.